### PR TITLE
Bootstrap meet_balls_app & add routing for webpage

### LINKED ISSUE #9: meet_balls_app is missing, and a web page must be created as well

### PR DESCRIPTION
**Pull request by Danielsio**
I created the templates folder contains the home.html file
edited the urls.py & views.py files to make the routing disaply the home page.

please review the changes for me to know if some fixes are needed.

please ignore the first commit beacuse it's only the stub i used to work on.

Depends on PR #7 

fixes #9

### CODE REVIEW

**Comment by Danielsio** [comment]
Also forgot to mention.. **DONT** merge until django enviroment PR is merged.

File: meet_balls_app/templates/meet_balls_app/home.html
Comment (yftacherzog) [inline meet_balls_app/templates/meet_balls_app/home.html:43]: i'm not sure if that goes hand in hand with the MIT license your repo has.

Comment (Danielsio) [reply meet_balls_app/templates/meet_balls_app/home.html:43]: Can you elaborate?

I can remove the copyright line..
Will that be enough?

Comment (yftacherzog) [reply meet_balls_app/templates/meet_balls_app/home.html:43]: I don't know enough about licensing, but I'd imagine that this line might imply that you want to reserve some rights. I think the MIT [LICENSE](https://github.com/redhat-beyond/MeetBalls/blob/main/LICENSE) file that you have at the root of the repo is more permissive and practically allows any use of your code.

All in all, yes. I think removing this line will make things simpler.

**Comment by Danielsio** [comment]
How can I edit the descreption once i already created this PR ?

**Comment by Yuval-Vino** [comment]
> How can I edit the descreption once i already created this PR ?

to edit a meesage on github click on the three dots(Kebab menu) located at the top of the message and select Edit

**Comment by Danielsio** [comment]
I added changes in the pr, now it has 3 commits, the first is the stub django enviroment.. 
the other 2 are the actual app called meet_balls_app
and the third is the update to the settings.py, and the urls.py to make the routing to the home.html.

i apologize for the discomfort

**Comment by yftacherzog** [comment]
@Danielsio having 3 commits on the PR should start you thinking of whether this PR is too big.
Larger PRs are harder to review and are also harder to fix. Imagine now that you need to fix any commit other than the last one. That quickly becomes a headache.

This will make it harder for your PR to be approved and merged.

I suggest:
1. Taking out of this PR anything which is not required for this task. Have MVP (minimum viable product) in mind. Do you need the styles to be able to deliver the first demo? If not, then get that change out. You can create another PR on top of it that will add the styles and can be merged if we get to it.
2. Once you decided on what's the minimum content you need to fulfill your task for the demo, squash the remaining content into as fewer commits as possible (preferably, one).
3. unrelated, verify that your commit messages follow the contributing guide. Specifically, the line about using the imperative mood.

**Comment by yftacherzog** [comment]
> > Also forgot to mention.. **DONT** merge until django enviroment PR is merged.
> 
> In addition to what Yftach said, Including the message "Depends on PR #7 " in the PR description will guarantee that the code is not merged until the Django environment PR has been successfully merged,

@Yuval-Vino  I'm not familiar with this feature. I'm not sure this is provided by GitHub out of the box.

@Danielsio I renamed the PR so it has the title of the commit this PR is about, and modified the description to include the dependency. It's not blocking the PR from being merged at the moment.

**Comment by OhadSeg** [comment]
@Danielsio I took a look into your commits, looks fine to me.

**Comment by amitKaplansky** [comment]
it looks good to me.

**Comment by MaayanMashhadi** [comment]
Looks good to me
Very well!

**Review by yftacherzog — APPROVED** [review]

**Pull request merged**
